.pre-commit-config.yaml

```diff
@@ -18,11 +18,6 @@ repos:
     hooks:
       - id: isort
         name: Format imports
-  - repo: https://github.com/MarcoGorelli/absolufy-imports
-    rev: v0.3.1
-    hooks:
-      - id: absolufy-imports
-        exclude: examples/directml/llm/chat_app/
```

Comment:
maybe keep this hook but exclude all of examples directory?

Reply:
https://github.com/microsoft/Olive/pull/1272/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R193

Xiaoyu already updated this.

Reply:
I meant to not delete this here. But just update the exclude part here to 'examples'. This way the pre commit hook can still fix the imports in other parts. 

Xiaoyu's change is for the lintrunner which is independent of the precommit hooks